<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopUp</class>
 <widget class="QWidget" name="PopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ramenRecipe">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="potatoRecipe">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="alfredoRecipe">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nachosRecipe">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="wrapRecipe">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="quesadillaRecipe">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="welcome">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome to College Cook! </string>
   </property>
  </widget>
  <widget class="QLabel" name="chooseRecipe">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CHOOSE A RECIPE TO START</string>
   </property>
  </widget>
  <widget class="QLabel" name="mealsInDorm">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>601</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Here's just a few of the gourmet meals you can make in your dorm!</string>
   </property>
  </widget>
  <widget class="QLabel" name="ramenLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RAMEN</string>
   </property>
  </widget>
  <widget class="QLabel" name="bakedPotatoLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BAKED POTATO</string>
   </property>
  </widget>
  <widget class="QLabel" name="alfredoLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ALFREDO</string>
   </property>
  </widget>
  <widget class="QLabel" name="nachosLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NACHOS</string>
   </property>
  </widget>
  <widget class="QLabel" name="chickenWrapLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CHICKEN WRAP</string>
   </property>
  </widget>
  <widget class="QLabel" name="quesadillaLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>QUESADILLA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
